src/site/content/en/react/performance-as-a-default-with-nextjs/index.md

```diff
@@ -0,0 +1,152 @@
+framework with a number of performance optimizations baked in. The main idea
+behind the framework is to ensure applications start and remain as performant
+as possible by having these capabilities included by default.
+
+This introduction will briefly cover many features provided by the framework
+at a high level. The other guides in this collection will explore the features
+in more detail.
+
+{% Aside %}
+  Chrome is collaborating with Next.js to improve the framework for any developer looking to
+  build a fast, server-rendered React application. A number of newer optimizations were recently
+  added such as [module/nomodule support](https://github.com/zeit/next.js/issues/7563) and an
+  [improved granular chunking strategy](https://github.com/zeit/next.js/issues/7631).
+{% endAside %}
+
+## What will you learn?
+
+Although Next.js provides a number of performance optimizations by default, these guides
+aim to explain them in more detail and show you how you can use them to build a fast and
+performant experience.
+
+{% Aside %}
+  This collection assumes that you have a basic knowledge of React. If not, check out 
+  [React's Getting Started page](https://reactjs.org/docs/getting-started.html).
+{% endAside %}
+
+There are many optimizations that can be added to React sites in general that would also
+work for applications built with Next.js. These will not be covered since the focus is
+on what Next.js specifically provides. To learn more about general React
+optimizations, check out our [React collection][collection].
+
+## How is Next.js different from React?
+
+React is a library that makes it easier to build user interfaces using a component-based
+approach. Although powerful, React is specifically a UI library. Many developers include
+additional tooling such as a module bundler ([webpack](https://webpack.js.org/) for example)
+and a transpiler ([Babel](https://babeljs.io/) for example) to have a complete build toolchain.
+
+In the [React collection][collection], we took the approach of using [Create React App](https://create-react-app.dev/)
+(CRA) to spin up React apps quickly. CRA takes the hassle out of setting up a React application
+by providing a complete build toolchain with a single command.
+
+Although there are a few default optimizations baked into CRA, the tool aims to provide a
+simple and straightforward setup. The choice is given to developers to decide whether
+to [eject](https://create-react-app.dev/docs/available-scripts#npm-run-eject) and
+modify the configurations themselves.
+
+Next.js, which can also be used to create a new React application, takes a different approach.
+It immediately provides a number of common optimizations that many developers would like to have
+but find difficult to set up, such as:
+
+* Server-side rendering
+* Automatic code-splitting
+* Route prefetching
+* File-system routing
+* CSS-in-JS styling (`styled-jsx`)
```

Comment:
should this link to https://github.com/zeit/styled-jsx ?

Reply:
Yes I believe the draft did originally and then I lost it while converting from Google Docs to Markdown. Another reason why we should author first drafts in Markdown within Google Docs.